From ffba2d32114b9b845d65f82dd6847fd14e29836b Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Tue, 22 Oct 2024 15:18:12 +0200
Subject: [PATCH 11/11] board: solidrun: imx8dxl-sr-som: enable emmc hardware
 reset

SolidRun i.MX8DXL SoM uses native usdhc1 emmc reset signal for eMMC
reset with dedicated pull-up resistor.

Add pinmux configuration selecting emmc reset function without internal
pull-up.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm/dts/fsl-imx8dxl-sr-som.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/dts/fsl-imx8dxl-sr-som.dts b/arch/arm/dts/fsl-imx8dxl-sr-som.dts
index 2868e39af8c..2a4b5109787 100644
--- a/arch/arm/dts/fsl-imx8dxl-sr-som.dts
+++ b/arch/arm/dts/fsl-imx8dxl-sr-som.dts
@@ -92,6 +92,7 @@
 				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000021
 				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000021
 				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x00000041
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000061
 			>;
 		};
 
@@ -108,6 +109,7 @@
 				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000021
 				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000021
 				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x00000041
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000061
 			>;
 		};
 
@@ -124,6 +126,7 @@
 				SC_P_EMMC0_DATA6_CONN_EMMC0_DATA6	0x00000021
 				SC_P_EMMC0_DATA7_CONN_EMMC0_DATA7	0x00000021
 				SC_P_EMMC0_STROBE_CONN_EMMC0_STROBE	0x00000041
+				SC_P_EMMC0_RESET_B_CONN_EMMC0_RESET_B	0x00000061
 			>;
 		};
 	};
-- 
2.43.0

